src/components/Sidebar/Sidebar.stories.tsx

```diff
@@ -35,7 +37,23 @@ const SidebarMeta = {
 } as ComponentMeta<typeof Sidebar>
 
 const Template: ComponentStory<typeof Sidebar> = () => {
-  return <Sidebar />
+  return (
+    <LoginContext.Provider
```

Comment:
nit: this isn't important but something to take note of is that such _wrapper_ components should be located in the `decorators` section of the meta (see [here](https://storybook.js.org/docs/react/writing-stories/decorators)) for more detail

Reply:
Right, thanks for sharing! Changed in https://github.com/isomerpages/isomercms-frontend/pull/959/commits/4f3601e3a2f3de92c0ab286766585057ea5073a4